ttnn/cpp/ttnn/operations/moreh/moreh_dot/device/moreh_dot_device_operation.hpp

```diff
@@ -57,12 +55,12 @@ struct MorehDotOperation {
     static tensor_return_value_t create_output_tensors(const operation_attributes_t&, const tensor_args_t&);
 
     static std::tuple<operation_attributes_t, tensor_args_t> invoke(
-        const Tensor &input_tensor_a,
-        const Tensor &input_tensor_b,
+        const Tensor& input_tensor_a,
+        const Tensor& input_tensor_b,
         const std::optional<DataType> output_dtype,
```

Comment:
use `const std::optional<DataType>& dtype` throughout until invoke
Note that `&` and `output_dtype` should be `dtype`